aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -342,6 +349,63 @@ module aptos_framework::account {
         rotation_capability
     }
 
+    /// Cffer signer capability to another address
+    public entry fun offer_signer_capability(
+        account: &signer,
+        signer_capability_sig_bytes: vector<u8>,
+        account_scheme: u8,
+        account_public_key_bytes: vector<u8>,
+        recipient_address: address
+    ) acquires Account {
+        let addr = signer::address_of(account);
+        assert!(exists_at(addr) && exists_at(recipient_address), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+
+        let account_resource = borrow_global_mut<Account>(addr);
+        // proof that this account intends to delegate its signer capability to another account
+        let signer_capability_offer_proof_challenge = SignerCapabilityOfferProofChallenge {
```

Comment:
Given that there's only one proof challenge in this function, the long name is not necessary: I'd just call this `proof_chal[lenge]`